src/SprykerEco/Zed/AmazonQuicksight/Business/ApiClient/AmazonQuicksightApiClient.php

```diff
@@ -46,6 +63,11 @@ class AmazonQuicksightApiClient implements AmazonQuicksightApiClientInterface
      */
     protected QuicksightUserMapperInterface $quicksightUserMapper;
 
+    /**
+     * @var \SprykerEco\Zed\AmazonQuicksight\Business\Mapper\QuicksightEmbedUrlMapperInterface
+     */
+    protected QuicksightEmbedUrlMapperInterface $quicksightEmbedUrlMapper;
```

Comment:
I think we'll need to merge mappers into one class eventually because we'll need to implement an asset bundle related functionality and we will need to add one more mapper.